Release/168.0.0 (#4469)
MetaMask/core: packages/notification-services-controller/CHANGELOG.md

## [Unreleased]

> ## [0.2.0]

Reviewer: If no features are being added and the bug fixes are the only new changes, maybe this should be a patch release?

```suggestion
## [0.1.1]
```

Author: OH, yes we can make this a patch release, no probs!

Author: TY for the comments, I've updated the changelog and packages for minor version.

> ## [0.1.1]
> 
> ### Changed
> 
> - `NotificationServicesController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))

Reviewer: Looks like there are feature additions here that might make this a minor release.

```suggestion
## [0.2.0]

### Added

- Export `NOTIFICATION_CHAINS_ID`, which is a const-asserted version of the `NOTIFICATION_CHAINS` object. ([#4441](https://github.com/MetaMask/core/pull/4441))
- Export const-asserted objects `NOTIFICATION_NETWORK_CURRENCY_NAME`, `NOTIFICATION_NETWORK_CURRENCY_SYMBOL`. ([#4441](https://github.com/MetaMask/core/pull/4441))
- Export `defaultState` object from `NotificationServicesPushController` ([#4441](https://github.com/MetaMask/core/pull/4441))

### Fixed

- `NotificationServicesController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

- Could we expand the "Fixed" entry to explain what's being fixed and any user-facing behavior changes?
  - Does the logic around `#isPushIntegrated` cause user-facing changes?
  - Is the replacement of `sw` with `self` a user-facing change?
- Is the removal of exports from `mockServices` user-facing? If so, it should be listed under a `### Removed` heading.

Author: TY for comments, I've added a more descriptive changelog

> ### Changed
> 
> - fix: notification services controller mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))

Reviewer: ```suggestion
### Fixed

- `NotificationServicesController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Author: TY, new to this repo!

Author: Yep good callout, I've included the Controller name, to make it a bit more explicit on the changes.

## [Unreleased]

## [0.1.1]

### Added

- export `defaultState` for `NotificationServicesController` and `NotificationServicesPushController`. ([#4441](https://github.com/MetaMask/core/pull/4441))

- export `NOTIFICATION_CHAINS_ID` which is a const-asserted version of `NOTIFICATION_CHAINS` ([#4441](https://github.com/MetaMask/core/pull/4441))

- export `NOTIFICATION_NETWORK_CURRENCY_NAME` and `NOTIFICATION_NETWORK_CURRENCY_SYMBOL`. Allows consistent currency names and symbols for supported notification services ([#4441](https://github.com/MetaMask/core/pull/4441))

> - added `isPushIntegrated` optional env property (defaults to true) ([#4441](https://github.com/MetaMask/core/pull/4441))

Reviewer: Looks good! Just had one last addition for clarity:
```suggestion
- add `isPushIntegrated` as an optional env property in the `NotificationServicesController` constructor (defaults to true) ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Author: Merged!

Author: Silly question - but do we have a changelog "best practices" doc or notes? E.g. past tense, be descriptive, etc

Reviewer: No that's a good point. Closest thing we have is this: https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md (especially section 5), but it could definitely be more comprehensive.
